### DISCUSSION TITLE
Is vue-node-registry support vue2?

### DISCUSSION THREAD

**Discussion by irelance** [didi/LogicFlow#1908] [Q&A] [answered]
Is vue-node-registry support vue2. I found that if I use Label extension with vue $mount has some problem. Such as label can't move. I want to try if the vue-node-registry can solve this problem.

**Comment by irelance** [comment]
Now I found that the official page has the same problem.
The Label sometimes can't move at all.

https://site.logic-flow.cn/tutorial/extension/label

**Comment by boyongjiong** [comment, marked as answer]
Vue2 is supported, but the component libraries bound to Vue cannot be used directly, there are some technical difficulties that we're trying to work out.  

Label doesn't move after double-clicking is a bug, I'll try to reproduce it and fix it. I don't think it has anything to do with Vue2 at the moment.

There are The Label Extension
